CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * Fix compiler warnings with MacOS Clang 14.0.3 ([PR #6467](https://github.com/realm/realm-core/pull/6467))
 * Perform a client reset to migrate a sync'd realm from PBS to FLX and vice versa ([#6393](https://github.com/realm/realm-core/issues/6393))
 * The following unused util headers have been deleted: call_with_tuple.hpp, get_file_size.hpp, inspect.hpp, substitute.hpp, type_list.hpp, and utf8.hpp.
+* Add migrated state information to flexible sync client BIND message ([PR #6464](https://github.com/realm/realm-core/pull/6464))
```

Comment:
Can we add something about how this will support the server back-filling partition values that aren't in your schema for migrated PBS realms? Just something so SDKs/customers know how this change actually effects them?

Reply:
Sure